### PR TITLE
Fix: Edit a task

### PR DESCRIPTION
**Pull request by shobhi1310**
## Description
Add modal and edit facility for user to edit the saved to-do tasks.

## Issues fixed
fixes: #1

### CODE REVIEW

File: dist/assets/css/index.css [hunk 1]
```diff
@@ -84,4 +84,68 @@ body {
...
+.close:hover,
+.close:focus {
+  color: black;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+/* Add Animation */
+@-webkit-keyframes animatetop {
+  from {top:-300px; opacity:0}
```
Comment (DevTony101) [inline dist/assets/css/index.css:144]: This should be in separate lines

File: dist/assets/css/index.css [hunk 2]
```diff
@@ -84,4 +84,68 @@ body {
...
+  cursor: pointer;
+}
+
+/* Add Animation */
+@-webkit-keyframes animatetop {
+  from {top:-300px; opacity:0} 
+  to {top:0; opacity:1}
+}
+
+@keyframes animatetop {
```
Comment (DevTony101) [inline dist/assets/css/index.css:148]: This should be in separate lines

**Review by DevTony101 — COMMENTED** [review]
Excellent work! My only complaint is that:

- The `edit` button should be on the right side of the content panel, so that it not add extra vertical space

**Comment by shobhi1310** [comment]
Hi @DevTony101 
I am Shubhankar.

I had made a PR in your repo last year under Hacktoberfest.
Actually, I am participating in GSoC this year. So the organization in which I am participating they are asking for a person who was either a mentor or a code maintainer where I had made earlier contributions.

Can I put your contact details as a referral?

**Pull request merged**